app/models/physical_ticket.rb

```diff
@@ -7,6 +7,12 @@ class PhysicalTicket < ActiveRecord::Base
 
   before_create :set_token
 
+  def mark_attendance
```

Comment:
I think that this method should belong to the User class no to this one. I mean, is changing a registration of a user, that's user's business. You can move this code there and delegate this method here to `user.mark_attendance_for(conference)`

Reply:
:+1